src/Annotation/VersionConstraints.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JMS\Serializer\Annotation;
+
+use Composer\Semver\Semver;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ */
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY)]
+final class VersionConstraints extends Version
+{
+    public function __construct($values = [], ?string $version = null)
+    {
+        if (!class_exists(Semver::class)) {
```

Comment:
what if this check is done in the metadata drivers? so we avoid to call it at runtime each time the annotation is instantiated?